### DISCUSSION TITLE
Update not working

### DISCUSSION THREAD

**Discussion by lancewicz** [roleoroleo/yi-hack-Allwinner-v2#786] [General]
Is there any issue that cameras don't see update to download and upgrade?
When I click upgrade fimrware I get an error

Unable to download firmware file.

**Comment by lancewicz** [comment]
I assume there is a broken link to newest FW.
Where there link is hosted? Is inside github or some external server?

kind regards,

**Reply by roleoroleo** [reply]
This is a known bug.
https://github.com/roleoroleo/yi-hack-MStar/issues/526
